tests/performance/hashed_dictionary.xml

```diff
@@ -48,91 +76,67 @@
             value_string_nullable Nullable(String)
         )
         PRIMARY KEY id, id_key
-        SOURCE(CLICKHOUSE(DB 'default' TABLE 'complex_key_hashed_dictionary_source_table'))
-        LAYOUT(COMPLEX_KEY_HASHED())
-        LIFETIME(MIN 0 MAX 1000);
+        SOURCE(CLICKHOUSE(TABLE 'complex_key_dictionary_source_table'))
+        LAYOUT(COMPLEX_KEY_{layout_suffix}())
+        LIFETIME(0)
     </create_query>
 
     <fill_query>
-        INSERT INTO simple_key_hashed_dictionary_source_table
+        INSERT INTO simple_key_dictionary_source_table
         SELECT number, number, toString(number), toDecimal64(number, 8), toString(number)
         FROM system.numbers
-        LIMIT 5000000;
+        LIMIT 5000000
     </fill_query>
 
     <fill_query>
-        INSERT INTO complex_key_hashed_dictionary_source_table
+        INSERT INTO complex_key_dictionary_source_table
         SELECT number, toString(number), number, toString(number), toDecimal64(number, 8), toString(number)
         FROM system.numbers
-        LIMIT 5000000;
+        LIMIT 5000000
     </fill_query>
 
-    <substitutions>
-        <substitution>
-            <name>column_name</name>
-            <values>
-                <value>'value_int'</value>
-                <value>'value_string'</value>
-                <value>'value_decimal'</value>
-                <value>'value_string_nullable'</value>
-            </values>
-        </substitution>
-
-        <substitution>
-            <name>elements_count</name>
-            <values>
-                <value>5000000</value>
-                <value>7500000</value>
-            </values>
-        </substitution>
-    </substitutions>
+    <fill_query>SYSTEM RELOAD DICTIONARY simple_key_{layout_suffix}_dictionary</fill_query>
+    <fill_query>SYSTEM RELOAD DICTIONARY complex_key_{layout_suffix}_dictionary</fill_query>
```

Comment:
That was the intention to measure `SYSTEM RELOAD` too, maybe it worth creating separate table for `sparse_hashed` table with smaller amount of elements (since it is indeed slower then `hashed`)?

Reply:
It is better to create separate tables for sparse_hashed_dictionary with 5-10 times less elements. I probably did not check performance tests carefully in pull request in the first place.
CC: @nickitat

Reply:
@nickitat that's true.

You can simply copy-paste everything instead.